charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+        name: http
+        protocol: TCP
+        targetPort: 8080
+    type: ClusterIP
+  ingress:
+    enabled: false
+    ingressClassName: nginx
+    servicePort: http
+    pathType: Prefix
+
+    # deviation from standard way of defining the services
+    hosts: []
+    # - host: backend.site.com
+    #   paths:
+    #     - path: /
+    #       pathType: Prefix
+    #       serviceName: backend
+    #       servicePort: http
+    additionalLabels: {}
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-staging
+      nginx.ingress.kubernetes.io/proxy-body-size: 100m
+      nginx.ingress.kubernetes.io/proxy-connect-timeout: '300'
+      nginx.ingress.kubernetes.io/proxy-read-timeout: '300'
+      nginx.ingress.kubernetes.io/server-snippet: |
+        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
+        # this prevents hidden files (beginning with a period) from being served
+        location ~ /\. {
+          access_log        off;
+          log_not_found     off;
+          deny              all;
+        }
+    tls: []
+    # - hosts:
+    #   - nginx.site.com
+    #   secretName: nginx.site.com-crt
+  pdb:
+    enabled: false
+    minAvailable: 1
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 2
+    # you can also adjust the value based on average actual value (regardless of the % use of the requested memory resources)
+    # for example:
+    #
+    # metrics:
+    # - type: Resource
+    #   resource:
+    #     name: memory
+    #     target:
+    #       type: AverageValue
+    #       averageValue: 900Mi
+    #
+    # see explanation: https://stackoverflow.com/questions/69184304/how-memory-metric-is-evaluated-by-kubernetes-hpa
+    metrics:
+      - type: Resource
+        resource:
+          name: cpu
+          target:
+            type: Utilization
+            averageUtilization: 70
+      - type: Resource
+        resource:
+          name: memory
+          target:
+            type: Utilization
+            averageUtilization: 70
+    behavior:
+      scaleDown:
+        stabilizationWindowSeconds: 300
+        policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 100
+          - type: Percent
+            value: 10
+            periodSeconds: 100
+      scaleUp:
+        stabilizationWindowSeconds: 180
+        policies:
+          - type: Pods
+            value: 1
+            periodSeconds: 30
+          - type: Percent
+            value: 10
+            periodSeconds: 60
+        selectPolicy: Max
+  configMap:
+    enabled: true
+    files:
+      config:
+        config.yaml: |
+          # Get available products from:
+          # https://endoflife.date/api/all.json
+          # and find available cycles in:
+          # https://endoflife.date/api/{product}.json
+          eks:
+            current: '1.30'
+            comment: EKS
+          django:
+            current: '5.1'
+            comment: backend
```

Comment:
don't define here, add it as comment, but define actual configmap in the actual cluster where this is deployed.

Reply:
and add documentation how is this working.

Reply:
Added